src/routes/PetsPage/PetsPage.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import PetCard from '../../components/PetCard/PetCard';
+import PetsService from '../../services/petsService';
+
+export default class PetsPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { pets: [] };
+  }
+
+  componentDidMount() {
+    PetsService.getPets()
+      .then(pets => this.setState({ pets }))
+      .catch(error => console.log(error));
+  }
+
+  renderPetRow(rowStart) {
+    return (
+      <div className='row'>
```

Comment:
I think you might need `key={rowStart} ` or something unique to avoid `unique "key" prop` warning here. Reference: https://reactjs.org/link/warning-keys. I can see that you handled the other sections. I think you just forgot this one :)